app/controllers/api/v1/driver_controller.rb

```diff
@@ -30,13 +30,27 @@ def index
     end
   end
 
+  def update_location
```

Comment:
@ameseee in Mod 3 we were taught to only have create, new, update, delete in our controller.  In my mind this method name makes sense though, I don't want a single update method handling the updating of both the location and the destination.  Any thoughts?